source/jormungandr/jormungandr/parking_space_availability/bss/stands.py

```diff
@@ -27,17 +27,22 @@
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
 from __future__ import absolute_import, print_function, unicode_literals, division
+from enum import Enum
 
-status_list = {'OPEN': 'open', 'CLOSED': 'closed', 'UNAVAILABLE': 'unavailable'}
-
+class StandsStatus(Enum):
+    unavailable = -1
```

Comment:
i'm just curious, why starting from -1?

Reply:
No reason, just tried to have "meaningful" values for all and -1 looked like a good one for "problem value".
But if you come to ask yourself why, I should maybe change that. No problem if you ask for it.